src/paymentservice/genproto.sh

```diff
@@ -1,6 +1,5 @@
-#!/bin/bash -eu
-#
-# Copyright 2018 Google LLC
+
```

Comment:
**Nitpick:**
There's a blank line before the start of each the license comment.
Was this intentional?

Reply:
this is how PyYaml throws the yaml content back into the file, with the license as a chunk of string
I don't mind removing them though!